backend/fms_core/serializers.py

```diff
@@ -116,7 +118,9 @@
     "MetricSerializer",
     "ArchivedCommentSerializer",
     "SampleIdentityMatchSerializer",
-    "SampleIdentitySerializer"
+    "SampleIdentitySerializer",
+    "FreezemanUserSerializer",
```

Comment:
don't make a FreezemanUserSerializer. This is a stub for connecting internal tables to django user table ...